OsmAnd/res/values/strings.xml

```diff
@@ -74,11 +74,11 @@
     <string name="years_2_4">Years</string>
     <string name="years_5">Years</string>
     <string name="months_3">Three months</string>
-    <string name="price_free">Free</string>
+    <string name="price_free">Gratis</string>
```

Comment:
I don't think it is commonly used english word. Though it is very common in dutch, german

Reply:
It is in common use, though in opposition to the more common misunderstanding between free/libre and gratis/freeware. The gratis version is both, so lets hope I don't forget again, and get to fixing more strings :)